napari/layers/labels/labels.py

```diff
@@ -476,6 +476,17 @@ def show_selected_label(self, filter):
     @property
     def brush_shape(self):
         """str: Paintbrush shape"""
+
+        if str(self._brush_shape) == 'square':
```

Comment:
I think I would show the warning always, as it will just be assumed to be circle, so we won't need the brush shape attribute at all

Reply:
Maybe also actually put the warning on the setter too so if someone tries to set they will get one too

Reply:
Ok, I remove that if statement and added the warning to the setter.  How is the wording of the warning?

Reply:
Thanks! How about adding on

```
"Afterward, only the circle brush shape will be available, and the `layer.brush_shape attribute will be removed"
```

That makes it clear we'll be taking away the attribute too